### DISCUSSION TITLE
5.0.0

### DISCUSSION THREAD

**Discussion by adrianhall** [Azure/azure-mobile-apps#332] [General]
* GA release of the new codebase.

<hr /><em>This discussion was created from the release <a href='https://github.com/Azure/azure-mobile-apps/releases/tag/5.0.0'>5.0.0</a>.</em>

**Comment by fileman** [comment]
in readme you use q=Microsoft.AspNetCore.Datasync in nuget link, but this dosent show client's pkgs, using q=Microsoft+Datasync show all pkgs

**Reply by adrianhall** [reply]
I like the PR.  However, they are all released by the same pipeline, so there isn't a difference.
